function.php

```diff
@@ -0,0 +1,107 @@
+<?php
```

Comment:
File contains multiple functions so functions.php is a better name

Reply:
true, changed